Code/C++/Union_of_two_unsorted_array.cpp

```diff
@@ -0,0 +1,61 @@
+// CPP program to find the Union of two unsorted arrays.
+
+/* Procedure 
+==================
+First of all here we'll initialize an empty hash set 's'.
+then we'll iterate through the first array and put every element of the first array in the set s.
+Repeat the process for the second array.
+Finally we'll print the set s.
+Time Complexity: O(m+n)
```

Comment:
This is not O(m+n) as you are using set, for inserting something in set it takes O(log n) time complexity.

So, please update it accordingly and shift this to the bottom of the code.